testdata/baselines/reference/submoduleAccepted/compiler/jsdocArrayObjectPromiseImplicitAny.errors.txt.diff

```diff
@@ -0,0 +1,74 @@
+--- old.jsdocArrayObjectPromiseImplicitAny.errors.txt
++++ new.jsdocArrayObjectPromiseImplicitAny.errors.txt
+@@= skipped -0, +-1 lines =@@
+-<no content>
+@@= skipped --1, +1 lines =@@
++jsdocArrayObjectPromiseImplicitAny.js(1,12): error TS2314: Generic type 'T[]' requires 1 type argument(s).
```

Comment:
Seems like some checking here isn't working?

Reply:
Seems like it's intentional due to `Array` meaning actual `Array` and not `evolving array`.

Reply:
I found very few examples of the synonyms `String -> string`, `Array -> Array<any>`, etc when I looked, and over the years I've had quite a few complaints about them from heavy JSDoc users, especially `object`. So they're not supported anymore.

This includes the syntax for index signatures `Object.<K,V> -> { [x: K]: V }`, which we ran into elsewhere in the diffs.

I'll add these files to submoduleAccepted.txt when the list of diffs gets smaller.